workflows/Purge_Office_365_Emails_with_Slack/workflow.spec.yaml

```
@@ -2,14 +2,14 @@ extension: workflow
 products: ["insightconnect"]
 name: Purge_Office_365_Emails_with_Slack
 title: "Purge Office 365 Emails with Slack"
-description: "This workflow will purge all emails in an organization matching given search criteria by sending a notification to InsightConnect with Slack."
-version: 1.0.0
+description: "This workflow uses a Slack command to purge all emails in an organization matching the provided search criteria."
```

```suggestion
description: "Use a Slack command to purge all emails in an organization matching provided search criteria."
```
Given how this displays in the hub, I think this will read better.

---

I agree but will need to update them all in one swoop